spec/support/ghost/ghost_stubs.rb

```diff
@@ -19,14 +19,6 @@ def stub_cms_page
       .to_return(body: raw_page_json)
   end
 
-  def stub_nested_cms_page
-    raw_page_json = File.new('spec/support/ghost/page.json')
-    stub_request(:get,
-                 "#{ENV['GHOST_API_ENDPOINT']}/content/pages/slug/subject-practitioners-primary/")
-      .with(query: hash_including({ 'key' => (ENV['GHOST_CONTENT_API_KEY']).to_s }))
-      .to_return(body: raw_page_json)
-  end
-
```

Comment:
I think we still need this example: we should probably use another page

Reply:
> Good changes. Can you find out if we still need the nested CMS page feature, and depending on what you find, follow up my notes below?

Message sent to Nikki the stackholder to get clarity on this

Reply:
> > Good changes. Can you find out if we still need the nested CMS page feature, and depending on what you find, follow up my notes below?
> 
> Message sent to Nikki the stackholder to get clarity on this

@dcorking I think we are good to take this off too
<img width="1116" alt="Screenshot 2023-05-22 at 13 40 13" src="https://github.com/NCCE/teachcomputing.org/assets/14819641/f7effd00-9b92-4221-b406-1884ae49e064">